tests/test_office_structure/test_add_node.py

```diff
@@ -0,0 +1,43 @@
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.office_structure.create_node_fixtures import (
+    node_creation_mutation,
+    duplicate_node_creation,
+    child_node_creation_mutation_with_non_existent_parent_id
+)
+
+
+class TestAddNode(BaseTestCase):
+
+    def test_add_node(self):
+        """
+        Tests whether an admin can node create a node
+        """
+
+        CommonTestCases.admin_token_assert_in(
+            self,
+            node_creation_mutation,
+            "floors"
+        )
+
+    def test_add_duplicate_node(self):
+        """
+        Tests whether an admin can create two nodes
+        with the same node
+        """
+        CommonTestCases.admin_token_assert_in(
+            self,
+            duplicate_node_creation,
+            "location node level already exists"
+        )
+
+    def test_add_child_node_with_non_existent_parent_id(self):
+        """
+        Tests whether an admin can create
```

Comment:
We are not testing whether this can happen, we are testing that this cannot happen

Reply:
This has been modified @Bonifase